react/src/components/atoms/forms/InputTextFuzzy/index.js

```diff
@@ -20,16 +20,70 @@ class InputTextFuzzy extends React.Component {
     fuseOptions.keys = this.props.keys;
     this.fuse = new Fuse(this.props.options, fuseOptions);
   }
-  handleChange = (e) => {
-    const suggestions = this.fuse.search(e.target.value);
+  clearSuggestions = (event) => {
     this.setState({
-      value: e.target.value,
-      suggestions
+      suggestions: [],
+      highlightedItemIndex: null
+    }, () => {
+      if (is.fn(this.props.onSuggestionClick)) {
+        this.props.onSuggestionClick(event, {
```

Comment:
i don't think we should pass this callback in blur

Reply:
onSuggestionClick is when you select on a suggestion, e.g. we use it to navigate to a url and that shouldn't be triggered onBlur